### PR TITLE
SCRUM-2409 SCRUM-2374 use diseaseRelationNegation field for filtering and creating the full value set

### PR DESCRIPTION
**Pull request by cmpich**

### CODE REVIEW

File: agr_api/src/main/java/org/alliancegenome/api/service/DiseaseESService.java [hunk 1]
```diff
@@ -100,6 +100,12 @@ public JsonResultResponse<GeneDiseaseAnnotationDocument> getRibbonDiseaseAnnotat
 					String[] elements = filterValue.split("\\|");
 					Arrays.stream(elements).forEach(element -> orClause.should(QueryBuilders.termQuery(filterName, element)));
 					bool.must(orClause);
+				} else if (filterValue.contains(" ")) {
```
Comment (oblodgett) [inline agr_api/src/main/java/org/alliancegenome/api/service/DiseaseESService.java:103]: @cmpich im not sure this is a good check... I think we can handle filtering better.

Comment (cmpich) [reply agr_api/src/main/java/org/alliancegenome/api/service/DiseaseESService.java:103]: I need to take each individual word and check for any contains matches. How can we do better?

Comment (oblodgett) [reply agr_api/src/main/java/org/alliancegenome/api/service/DiseaseESService.java:103]: Use a simple query and the "AND" operator

Comment (oblodgett) [reply agr_api/src/main/java/org/alliancegenome/api/service/DiseaseESService.java:103]: Or couldn't you just change the spaces to underscores and then you would have an exact match?

Comment (cmpich) [reply agr_api/src/main/java/org/alliancegenome/api/service/DiseaseESService.java:103]: I don't want an exact match. 
<query1> <query2> <query3>
should be interpreted as
<*query1*>AND<*query2*>AND<*query3*>
ie as wildcard matches for each query string connected with AND.

Comment (oblodgett) [reply agr_api/src/main/java/org/alliancegenome/api/service/DiseaseESService.java:103]: Which field is this for?

Comment (cmpich) [reply agr_api/src/main/java/org/alliancegenome/api/service/DiseaseESService.java:103]: Disease name

**Pull request closed without merging**